ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestQueryTranslationTest.java

```diff
@@ -108,6 +108,7 @@
       .withProperty(KsqlConfig.KSQL_QUERY_PULL_INTERPRETER_ENABLED, true)
       .withProperty(KsqlConfig.KSQL_QUERY_PUSH_V2_REGISTRY_INSTALLED, true)
       .withProperty(KsqlConfig.KSQL_QUERY_PUSH_V2_ENABLED, true)
+      .withProperty(KsqlConfig.KSQL_ROWPARTITION_ROWOFFSET_ENABLED, true)
```

Comment:
This does not look like it's meant to be checked in.

Reply:
Well, I added it to verify that this works with the new feature enabled.  I could avoid checking it in, but then something could break again and we wouldn't know.  I'm ok to enable it by default here unless you think it's not a good idea.

Reply:
OK, if the RQTTs are passing with this enabled then I'm happy enabling it if you are. We're planning to flip the feature flag default to true within the next week anyway.